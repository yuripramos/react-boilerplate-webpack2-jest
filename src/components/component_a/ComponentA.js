import React from "react";
import style from "./style.scss";

const ComponentA = ({ name }) => {
	// debugger;
	return (
		<div className={`${style.container} row`} >
			<div className="col-lg-12 col-md-12 col-sm-12">
				<p> {`Hey ${name} I'm your Component A`}</p>
			</div>
		</div>
	)
};

export default ComponentA;
