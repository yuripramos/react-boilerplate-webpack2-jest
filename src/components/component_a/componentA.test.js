import React from 'react';
import renderer from 'react-test-renderer';

import ComponentA from './ComponentA.js';

describe('rendering tests', () => {
	it('renders correctly', () => {
		const component = renderer.create(<ComponentA />);
		const tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
