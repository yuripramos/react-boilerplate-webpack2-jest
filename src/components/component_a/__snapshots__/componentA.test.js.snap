// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering tests renders correctly 1`] = `
<div
  className="container row"
>
  <div
    className="col-lg-12 col-md-12 col-sm-12"
  >
    <p>
       
      Hey undefined I'm your Component A
    </p>
  </div>
</div>
`;
